index.html

```diff
@@ -136,9 +136,9 @@
         <div class="download-section">
             <div class="container">
                 <div class="col-lg-8 col-lg-offset-2">
-                    <h2>Come and watch them fight with their keyboard.</h2>
-                    <p>Chose your side and support your team.</p>
-                    <a href="http://meetup.com/fightcode/" class="btn btn-default btn-lg">Subscribe</a>
+                    <h2>Come and watch the first Angular 2 coding fight.</h2>
+                    <p>Choose the right side and support your team.</p>
+                    <a href="http://meetup.com/fightcode/" class="btn btn-default btn-lg">Let the fight begin</a>
```

Comment:
@tagazok En lisant les correctifs de FX je pensais que "Let the fight begin" était le nouveau texte du CTA, qu'en pensez-vous ?